12:52:06 AM: DEBUG [logger_base.py:11] Mensaje a nivel debug
12:52:06 AM: INFO [logger_base.py:12] Mensaje nivel info
12:52:06 AM: WARNING [logger_base.py:13] Mensaje nivel warning
12:52:06 AM: ERROR [logger_base.py:14] Mensaje nivel error
12:52:06 AM: CRITICAL [logger_base.py:15] Mensaje nivel critico
02:07:27 AM: DEBUG [conexion.py:23] Conexión exitosa: <connection object at 0x00000275CE1EC480; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
02:07:27 AM: DEBUG [conexion.py:35] Se abrio correctamente el cursor: <cursor object at 0x00000275CFD6CF20; closed: 0>
02:07:27 AM: DEBUG [persona_dao.py:25] 
        Id Persona: 1, Nombre: Juan,
        Apellido: Caisapanta, Email: jcaisapanta@mail.com

